src/style.css

```diff
@@ -0,0 +1,156 @@
+@font-face {
+  font-family: Roboto;
```

Comment:
Regarding the blocks, since this is a BEM task, I would recommend creating new .css files for each of your blocks.